.github/workflows/quemb_unittest.yml

```diff
@@ -116,7 +116,7 @@ jobs:
     - name: Test with pytest
       run: |
         cd tests
-        QUEMB_SKIP_EXPENSIVE_TESTS=true pytest --doctest-modules --junitxml=junit/quemb-test-results_${{ matrix.python-version }}.xml
+        QUEMB_SKIP_EXPENSIVE_TESTS=true pytest --durations=0 --durations-min=1.0 --doctest-modules --junitxml=junit/quemb-test-results_${{ matrix.python-version }}.xml
```

Comment:
isn't the minimum `--durations-min` redundant, if you don't report any durations?

Reply:
Oh yeah I also find this counterintuitive lol pytest reports `n` longest tests normally. If `n` is zero, it reports all tests that took longer than what was given with `durations-min`

Reply:
And just an aside, `durations-min` takes values in seconds haha min probably stands for minimum, not minutes

Reply:
ok then the change makes sense. 
I would perhaps add a comment explaining that `--durations=0` is indeed what we want and not an oversight.

Reply:
Sure will do!